frontend/tests/unit/models/transportation-analysis-test.js

```diff
@@ -1,13 +1,66 @@
 import { module, test } from 'qunit';
 import { setupTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 
 module('Unit | Model | transportation analysis', function(hooks) {
   setupTest(hooks);
+  setupMirage(hooks);
 
-  // Replace this with your real tests.
-  test('it exists', function(assert) {
-    let store = this.owner.lookup('service:store');
-    let model = store.createRecord('transportation-analysis', {});
-    assert.ok(model);
+  test('it correctly calculates square feet and ratios for land use', async function(assert) {
+    let analysisMirage = server.create('transportation-analysis', 'projectLandUse');
+    let project = await this.owner.lookup('service:store').findRecord(
+      'project', analysisMirage.projectId, { include: 'transportation-analysis' }
+    );
+    let analysis = await project.get('transportationAnalysis');
+
+    assert.equal(analysis.residentialUnits, 1000) // project.totalUnits
```

Comment:
I'm not loving that the original data values that are ultimately being asserted through this test are not explicit in the test setup. If someone changes the default transportation-analysis model data, your tests fail. Might be worth explicitly creating the model w/ the values under test here in the test itself.